server.go

```diff
@@ -47,8 +46,7 @@ var (
 )
 
 // newServer returns a new language server.
-func newServer(client protocol.ClientCloser) (*server, error) {
-	jpaths := filepath.SplitList(os.Getenv("JSONNET_PATH"))
+func newServer(client protocol.ClientCloser, jpaths []string) (*server, error) {
```

Comment:
nit: I think `jpath` doesn't need to be plural.

Reply:
Oh I realized while manually implementing the this that it is plural in the go-jsonnet package :D